src/screens/splash/index.tsx

```diff
@@ -42,9 +46,19 @@ export default function SplashScreen({ navigation }: SplashScreenProp) {
     ScreenGridSizeActions(dispatch, CARD_ITEM);
   };
 
-  const checkInitialLaunch = async () => {
+  const checkInitialLaunch =  () => {
     // do checks here for initial launch and subsequent launch
-    navigation.replace('HomeScreen');
+    useLocalCustom().then(load =>{
+       if (!load.firstTime){
+        setFirst_Time_Application_load();
+       }
+         console.log('loading happening', load); 
+        load.remember
+          ? navigation.replace('HomeScreen')
```

Comment:
this structure of ternary is too complex for project teammates
refactor to a more readable approach  (Ternary should be used for only one single if-else statement )